src/ansys/aedt/core/maxwell.py

```diff
@@ -3179,6 +3179,50 @@ def assign_resistive_sheet(
 
         return self._create_boundary(name, props, "ResistiveSheet")
 
+    def order_coil_terminals(self, winding_name=None, list_of_terminals=None):
+        """Order coil terminals
+
+        Create custom connection order amongst different turns in a Winding definition.
+        Note that this feature is only available in the A-Phi formulation.
+
+        Parameters
+        ----------
+        winding_name : str
+            Name of Winding in AEDT
```

Comment:
```suggestion
        winding_name : str, optional
            Name of winding in AEDT.
            If no name is provided a random name will be generated.
```
In the implementation you are not handling the case when winding_name = None. If it's none you should use the generate_unique_name("Winding") or whichever prefix you prefer.

Reply:
@gmalinve, to use the feature both parameters winding and coil terminals need to be defined. Furthermore, the winding needs to be already populated with the terminals to be ordered. Otherwise, you can't use the feature. It would be possible to generate random names internally, but then the user wouldn't know what terminals or windings are available to them.